packages/web/src/scss/components/Accordion/_Accordion.scss

```
@@ -0,0 +1,97 @@
+@use 'sass:map';
+@use 'theme';
+@use '../Collapse';
+
+// .Accordion {}
+
+.Accordion__item + .Accordion__item {
+    margin-top: theme.$accordion-spacer;
+}
+
+.Accordion__itemHeader {
+    position: relative;
+    display: flex;
+    gap: theme.$accordion-spacer;
+    align-items: flex-start;
+    justify-content: space-between;
+    width: 100%;
+    height: auto;
+    padding: calc(#{theme.$accordion-spacer} * 2) theme.$accordion-spacer;
+    margin-bottom: 0;
+
+    &::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: theme.$accordion-spacer;
+        display: block;
+        width: calc(100% - (#{theme.$accordion-spacer} * 2));
+        height: 0;
+        border-bottom: 1px solid theme.$accordion-item-hover-color;
+    }
+
+    &:hover {
```

```suggestion
    &:hover::before {
```
Please avoid unnecessary nesting.